### ISSUE TITLE
Spotlight turn off when button released instead of move timeout

### ISSUE THREAD

**Issue by bgottschall** [jahnf/Projecteur#221]
**Is your feature request related to a problem? Please describe.**
I hold the spotlight button on the presenter and move the device which enables spotlight on the screen. If I hold the presenter very steady then spotlight is deactivated. 

**Describe the solution you'd like**
In my head it would make more sense to activate spotlight upon a long press on the button and only deactivate it when the long press is released. This way one can point on something on the screen and not worry that one needs to move the presenter so that spotlight doesn't turn off.

**Describe alternatives you've considered**
Right now one needs to always move the presenter to keep spotlight on?!

**Comment by jahnf** [comment]
Yes, the timeout was intentional, since for a lot of devices (also for the Logitech device if you don't reverse engineer their proprietary protocol) there is no way to detect if that button (that enables the mouse movement) is pressed or released.

That said, you can assign 'Toggle Spotlight' functionality to any button. Does this help your use case?

**Comment by abalg** [comment]
How about using a double-click to turn it on/off ?
